src/App.js

```diff
@@ -0,0 +1,464 @@
+                <Tab>Spells</Tab>
+                <Tab>Inventory</Tab>
+                <Tab>Levels</Tab>
+            </TabList>
+
+
+            <TabPanel>
+		<input type="submit" value="Account"/>
+		<input type="submit" value="Characters"/>
+		<input type="submit" value="Settings"/>
+		<input type="submit" value="Terms of Service"/>
+		<input type="submit" value="Help"/>
+            </TabPanel>
+            <TabPanel>
+            	<form>
+			<div>
+        			<label>
+          				Name:
+          				<input type="text" name="name"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          				Race:
+          				<input type="text" name="race"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          				Character Level:
+          				<input type="text" name="level"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          				Alignment:
+          				<input type="text" name="alignment"/>
+        			</label>
+			</div>
+      		</form>
+			<div>			
+				<p>Max Health:	</p>
+			</div>
+			<div>
+				<p>STR:	</p>
+			</div>
+			<div>
+				<p>DEX:	</p>
+			</div>
+			<div>
+				<p>CON:	</p>
+			</div>
+			<div>
+				<p>INT:	</p>
+			</div>
+			<div>
+				<p>WIS:	</p>
+			</div>
+			<div>
+				<p>CHA:	</p>
+			</div>
+
+				<p>Character Description:</p>
+				<textarea>
+ 			
+				</textarea>
+
+			<p>Backstory:</p>
+			<textarea>
+ 			
+			</textarea>
+            </TabPanel>
+
+            <TabPanel>
+
+			<div>
+        			<label>
+          				MAX HP:
+          				<input type="text" name="maxHP"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          				HP:
+          				<input type="text" name="hp"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          				AC:
+          				<input type="text" name="AC"/>
+        			</label>
+			</div>
+			<div>
+        			<label>
+          			Touch:
+          			<input type="text" name="Touch"/>
+        			</label>
+			</div>
+
+			<div>
+				<p>STR:	</p>
+			</div>
+			<div>
+				<p>DEX:	</p>
+			</div>
+			<div>
+				<p>CON:	</p>
+			</div>
+			<div>
+				<p>INT:	</p>
+			</div>
+			<div>
+				<p>WIS:	</p>
+			</div>
+			<div>
+				<p>CHA:	</p>
+			</div>
+
+			<div>
+				<p>Fortitude:	</p>
+			</div>
+			<div>
+				<p>Reflect:	</p>
+			</div>
+			<div>
+				<p>Will:	</p>
+			</div>
+
+
+			<div>
+				<p>Available Spells:</p>
+				<p>Spells that are avilable</p>
+			</div>
+
+			<div>
+				<p>Base Attack Bonus:	</p>
+			</div>
+			<div>
+				<p>Spell Ressistance:	</p>
+			</div>
+			<div>
+				<p>Flat Footed:	</p>
+			</div>
+			<div>
+				<p>CMD:	</p>
+			</div>
+			<div>
+				<p>Inititive:	</p>
+			</div>
+
+			<div>
+				<p>Abilities:</p>
+				<p>Abilities that are avilable</p>
+			</div>
+
+            </TabPanel>
+
+            <TabPanel>
+		<div>
+                	<input type="submit" value="-"/>
+				Acrobatics:  
+                	<input type="submit" value="+"/>
+		</div>
```

Comment:
Again, use the `<button>` tag. These should just be normal button type and not submit, and when the client goes into edit mode there will be a single submit button. Otherwise we have a ton of unnecessary network traffic every time a button is pressed.

Reply:
This applies to all of the skills. This is another good candidate for creating a custom React element.